objects/AllPlayerCards.15bb07/ZoeySamaras.98a0e1.gmnotes

```diff
@@ -7,5 +7,37 @@
   "intellectIcons": 2,
   "combatIcons": 4,
   "agilityIcons": 2,
-  "cycle": "The Dunwich Legacy"
+  "cycle": "The Dunwich Legacy",
+  "deck_requirements": {
+    "size": 30,
+    "randomBasicWeaknessCount": 1,
+    "signatures": [
+      {
+        "02006": "02006"
+      },
+      {
+        "02007": "02007"
+      }
+    ]
+  },
+  "deck_options": [
+    {
+      "faction": [
+        "guardian",
+        "neutral"
+      ],
+      "level": {
+        "min": 0,
+        "max": 5
+      }
+    },
+    {
+      "level": {
+        "min": 0,
+        "max": 0
+      },
+      "limit": 5,
+      "error": "You cannot have more than 5 cards that are not Guardian or Neutral"
```

Comment:
Is this going to work with Versatile?

Reply:
This doesn't do anything currently, this is more laying the groundwork for eventual deck validation.